### PR TITLE
FFT chapter edits in response to editor's feedback

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

File: markdown/ch4.markdown [hunk 1]
```diff
@@ -787,7 +899,8 @@ provide examples of various applications of the FFT. We will use actual
 data from an FMCW radar to demonstrate one such an application: target
 detection.
 
-Roughly[^detail_fmcw], an FMCW radar works like this:
+Roughly[^detail_fmcw], an FMCW radar works like this (see box "A
```
Comment (jni) [inline markdown/ch4.markdown]: @stefanv the subscript needs to go, right?

Comment (stefanv) [inline markdown/ch4.markdown]: Thanks, good catch.

File: markdown/ch4.markdown [hunk 2]
```diff
@@ -84,6 +86,8 @@ ax.set_xlabel('Time [s]')
 ax.set_ylabel('Signal amplitude');
 ```
```
Comment (jni) [inline markdown/ch4.markdown]: @hdashnow I did but in the interest of expedience, I'll make the changes myself this time. =)

**Comment by jni** [comment]
Merging to address @hdashnow's comments

**Pull request merged**